### ISSUE TITLE
blog post about choosing your repository implementation

### ISSUE THREAD

**Issue by jku** [theupdateframework/tuf#2452]
I'd like to have a text I can link to that explains some of the things that affect the choices in TUF repository setup and (now that we have multiple implementations) makes practical comparisons.

We should write a blog post that talks about 
* basic requirements that apply to any TUF setup (e.g. _trusted client software that you can embed root metadata in_)
* what tuf-on-ci and RSTUF are good for and and how they work differently

_Planning to set up a TUF repository to securely deliver your digital artifacts? Confused by all of the choices available? This essay is for you!_

@kairoaraujo a blog collaboration maybe?

**Comment by kairoaraujo** [comment]
@jku Yes, I would love to collaborate on this post. Thank you for the great opportunity.

**Comment by joshuagl** [comment]
I'd be happy to review early drafts
